README.md

```
@@ -1,40 +1,228 @@
+
+### GitHub Integration
+
+### Define the data repository
+
+1. Fork the repository:
+    - Visit http://github.com/ever-works/awesome-data
+    - Click "Fork" to create a copy
+    - This repo will hold `.content` data
+2. Configure GitHub integration:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)